musthe/musthe.py

```diff
@@ -332,6 +333,20 @@ class Chord:
         'ø7':     'm7dim5',
         'm7b5':   'm7dim5',
     }
+    
+    lilypond_modifiers = {
+        'maj': None,
+        'min': 'm',
+        'dom7': '7',
+        'min7': 'm7',
+        'm7dim5': '7.5-',
+        'open5':  '1.5.8',        
+    }
+    """Maps Musthe chord modifiers to their LilyPond `\chordmode` equivalents. Any items
+    omitted from this list (``min``, ``aug`` and ``dim``, for example) are
+    assumed to use the same value in LilyPond.
+    """
```

Comment:
can you move this comment above the variable declaration?

Reply:
Will do. (I was surprised to find that the "official" best practice for attribute docstrings is putting them after the attribute, but I'm not crazy about it either)

Reply:
This should be fixed in the updated pr.